pkg/collector/scheduler.go

```diff
@@ -147,6 +149,20 @@ func (s *CheckScheduler) getChecks(config integration.Config) ([]check.Check, er
 	return []check.Check{}, fmt.Errorf("unable to load any check from config '%s'", config.Name)
 }
 
+// GetChecksByNameForConfigs returns checks matching name for passed in configs
+func GetChecksByNameForConfigs(checkName string, configs []integration.Config) []check.Check {
```

Comment:
This method is not safe, there is no guarantee that the Scheduler has been initialised, you should check this first and return an error if it's `nil`.